cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/bridge_to_ethereum_config.rs

```diff
@@ -188,6 +190,8 @@ impl snowbridge_pallet_system::Config for Runtime {
 	type Helper = ();
 	type DefaultPricingParameters = Parameters;
 	type InboundDeliveryCost = EthereumInboundQueue;
+	type RegisterTokenOrigin =
+		EnsureOriginWithControlFlag<Runtime, EnableRegisterToken, TreasuryAccount>;
```

Comment:
does `TreasuryAccount` even have any funds on BridgeHub?

better to just make the tx free for Root rather than try to pay from Treasury account on BH